WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SiteCreationPreviewFragment.kt

```diff
@@ -72,6 +77,9 @@ class SiteCreationPreviewFragment : SiteCreationBaseFormFragment(),
 
     private var okButtonContainer: View? = null
 
+    // an instance helping sequence texts while in `loading` state
+    private var mTextsProgressTextsHelper: DynamicTextsProgressionHelper? = null
```

Comment:
We use the hungarian notation only in Java files. It'd be good to remove it for consistency. Thanks!